data-source/blocks/individual/identity-and-health/arrive_in_uk.jsonnet

```diff
@@ -23,7 +23,49 @@ local proxyTitle = {
   ],
 };
 
-{
+local routingRules(region_code) = [
+  {
+    goto: {
+      block: 'length-of-stay',
+      when: [
+        {
+          id: 'arrive-in-uk-answer',
+          condition: 'greater than',
+          date_comparison: {
+            value: std.extVar('census_date'),
+            offset_by: {
+              years: -1,
+            },
+          },
+        },
+      ],
+    },
+  },
+  {
+    goto: {
+      block: 'when-arrive-in-uk',
+      when: [
+        {
+          id: 'arrive-in-uk-answer',
+          condition: 'equals',
+          date_comparison: {
+            value: std.extVar('census_date'),
+            offset_by: {
+              years: -1,
+            },
+          },
+        },
+      ],
+    },
+  },
+  {
+    goto: {
+      block: if region_code == 'GB-WLS' then 'understand-welsh' else 'language',
+    },
+  },
+];
+
+function(region_code) {
```

Comment:
If region code was used in the same way as census_date this wouldn't need to be a function. What are the advantages of converting it to a function?

Reply:
Thought it would suit better to have a top level argument that is passed to only the places we need rather than having global var, but ya could use the std.Var.

According to Jsonnet:
>Generally, top-level arguments are the safer and easier way to parameterize an entire config, because the variables are not global and it's clear what parts of the config are dependent on their environment.

Reply:
Yeah, it is nice to make it clear what blocks are dependent on region code. I think we should change the census date to use the same thing so that they're consistent.

Reply:
So have changed census date to use the same thing but requires passing census_date into proxy titles etc, have a look and see.

Reply:
I think that's fine, we should look at moving the proxy text choice into the question in another pr.